TFLiteSwift-Vision/Classes/TFLiteVisionInterpreter.swift

```diff
@@ -198,18 +197,32 @@ public class TFLiteVisionInterpreter {
         return outputTensors.map { TFLiteFlatArray(tensor: $0) }
     }
     
-    public func inference(with uiImage: UIImage) -> TFLiteFlatArray<Float32>? {
+    public func inference(with uiImage: UIImage) -> [TFLiteFlatArray<Float32>]? {
         let input: TFLiteVisionInput = .uiImage(uiImage: uiImage)
         
         // preprocess
         guard let inputData: Data = preprocess(with: input)
             else { fatalError("Cannot preprcess") }
         
         // inference
-        guard let output: TFLiteFlatArray<Float32> = inference(with: inputData)?.first
+        guard let outputs: [TFLiteFlatArray<Float32>] = inference(with: inputData)
+            else { fatalError("Cannot inference") }
+        
+        return outputs
+    }
+    
+    public func inference(with pixelBuffer: CVPixelBuffer) -> [TFLiteFlatArray<Float32>]? {
+        let input: TFLiteVisionInput = .pixelBuffer(pixelBuffer: pixelBuffer)
+        
+        // preprocess
+        guard let inputData: Data = preprocess(with: input)
+            else { fatalError("Cannot preprcess") }
```

Comment:
How about throw error when something wrong with inference? fatalError is to let them know something wrong when developer test own app. But If an error occurs at runtime, they couldn't handle it.

Reply:
@Seonghun23 
nice idea, i'm gonna support that function in next implementation.
thanks for your feedback 💪